File: kernels/jvm/src/main/java/io/github/mosser/arduinoml/kernel/structural/COMPARATOR.java
package io.github.mosser.arduinoml.kernel.structural;

public enum COMPARATOR {
    EQ("=="),
    NEQ("!="),
    GT(">"),
    GEQ(">="),
    LT("<"),
    LEQ("<=");

    private String operator;

    COMPARATOR(String operator) {
        this.operator = operator;
    }

    public String getOperator() {
        return operator;
    }

    @Override
    public String toString() {
        return operator;
    }
}

File: embeddeds/python/pyArduinoML/model/COMPARATOR.py
from enum import Enum


class Comparator(Enum):
    """
    Enumeration of comparison operators.
    """
    EQ = "=="
    NEQ = "!="
    GT = ">"
    GEQ = ">="
    LT = "<"
    LEQ = "<="

    def __str__(self):
        """
        Returns the  string representation of the comparator.

        :return: str, the comparator as a string.
        """
        return self.value
